clang-tools-extra/clang-tidy/modernize/UseStartsEndsWithCheck.cpp

```diff
@@ -240,6 +240,9 @@ void UseStartsEndsWithCheck::check(const MatchFinder::MatchResult &Result) {
                                        ReplacementFunction->getName());
 
   // Replace arguments and everything after the function call.
+  if (FindExpr->getNumArgs() == 0) {
+    return;
+  }
```

Comment:
Technically all matchers that bind `find_expr` do bind on a call with more than one argument, but we can leave this in. IMO this should be moved before any fixes are added to the diagnostic, otherwise there will be a partial fix attached to the diagnostic. 

nit: single expr as *then* -> no braces around the return

Reply:
moved it before the warning. or should it go after the warning, but before the first fixit?